packages/indexer-common/src/indexer-management/resolvers/allocations.ts

```diff
@@ -0,0 +1,1250 @@
+  ): Promise<object[]> => {
+    logger.debug('Fetch allocations', {
+      filter,
+    })
+
+    const allocations: AllocationInfo[] = []
+
+    const currentEpoch = await contracts.epochManager.currentEpoch()
+    const disputeEpochs = await contracts.staking.channelDisputeEpochs()
+    const variables = {
+      indexer: toAddress(address),
+      disputableEpoch: currentEpoch.sub(disputeEpochs).toNumber(),
+      allocation: filter.allocation
+        ? filter.allocation === 'all'
+          ? null
+          : toAddress(filter.allocation)
+        : null,
+      status: filter.status,
+    }
+    const context = {
+      currentEpoch: currentEpoch.toNumber(),
+      currentEpochStartBlock: (
+        await contracts.epochManager.currentEpochBlock()
+      ).toNumber(),
+      currentEpochElapsedBlocks: (
+        await contracts.epochManager.currentEpochBlockSinceStart()
+      ).toNumber(),
+      latestBlock: (await contracts.epochManager.blockNum()).toNumber(),
+      maxAllocationEpochs: await contracts.staking.maxAllocationEpochs(),
+      blocksPerEpoch: (await contracts.epochManager.epochLength()).toNumber(),
+      avgBlockTime: 13_000,
+    }
+
+    allocations.push(
+      ...(await queryAllocations(logger, networkSubgraph, contracts, variables, context)),
+    )
+    logger.debug('Successfully fetched allocations', {
+      allocationsFetched: allocations.length,
+    })
+    return allocations
+  },
+
+  createAllocation: async (
+    { deploymentID, amount }: { deploymentID: string; amount: string },
+    {
+      address,
+      contracts,
+      subgraphManager,
+      logger,
+      models,
+      networkSubgraph,
+      transactionManager,
+    }: IndexerManagementResolverContext,
+  ): Promise<CreateAllocationResult> => {
+    logger.info('Creating allocation', { deploymentID, amount })
+
+    const allocationAmount = parseGRT(amount)
+    const subgraphDeployment = new SubgraphDeploymentID(deploymentID)
+    let activeAllocations: Allocation[] = []
+
+    // Fetch active allocations
+    try {
+      const result = await networkSubgraph.query(
+        gql`
+          query allocations($indexer: String!, $status: AllocationStatus!) {
+            allocations(where: { indexer: $indexer, status: $status }, first: 1000) {
+              id
+              indexer {
+                id
+              }
+              allocatedTokens
+              createdAtEpoch
+              closedAtEpoch
+              createdAtBlockHash
+              subgraphDeployment {
+                id
+                stakedTokens
+                signalledTokens
+              }
+            }
+          }
+        `,
+        {
+          indexer: address.toLocaleLowerCase(),
+          status: AllocationStatus[AllocationStatus.Active],
+        },
+      )
+
+      if (result.error) {
+        throw result.error
+      }
+      logger.info('Results', { result })
```

Comment:
Maybe we can specify the results and lower the level to `trace` or `debug`

Reply:
Thanks for pointing this out.  Looks like I left some debug logs in! Oops, I'll remove.